test/src/test/java/org/axonframework/test/server/AxonServerSEContainerTest.java

```diff
@@ -0,0 +1,30 @@
+package org.axonframework.test.server;
+
+import org.junit.jupiter.api.*;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * Simple test class for AxonServerSEContainer.
+ */
+class AxonServerSEContainerTest {
```

Comment:
Would it make sense to have tests for all the public methods on the `AxonServerSEContainer` too? :thinking:

Reply:
I will try to spend some time on it and see what can be asserted properly.